css/styles.css

```diff
@@ -202,11 +203,12 @@ figure.chuck-container {
 }
 
 .search-title {
-  margin-bottom: 5px;
+  margin-bottom: 25px;
```

Comment:
Everything should be in rems to make the site responsive now. Rather than putting a comment on each px size, I'll just leave the comment here. Thanks!

Reply:
@GillianTrethewey 
It looks like these two css elements were already set to px when I got the file (I just increased them a bit). I'll swap these over to rems now, and see what else I can adjust.
Thanks!